mandatory/2-logic-error.js

```diff
@@ -1,11 +1,11 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
 }
 
 function multiply(a, b, c) {
```

Comment:
Did you get a chance to look at this multiply function? It looks like you will still need to return a value here.

Reply:
Yes, now I fixed the errors